app/models/project.rb

```diff
@@ -2,6 +2,20 @@
 class Project < ActiveRecord::Base
   using SymbolRefinements
 
+  #  BADGE_STATUS_CHOICE = %w(in_progress passing failing).freeze
+  class BadgeStatus
+    attr_reader :id, :name
+
+    def initialize(id, name)
+      @id = id
+      @name = name
+    end
+  end
+
+  BADGE_STATUS_CHOICE = [BadgeStatus.new(nil, nil),
+                         BadgeStatus.new('in_progress', 'in progress'),
+                         BadgeStatus.new('passing', 'passing'),
+                         BadgeStatus.new('failing', 'failing')].freeze
```

Comment:
There's no need to create a BadgeStatus model. There are other Rails helpers that don't expect a model. So, this can just be:
```ruby
BADGE_STATUSES = (nil + %w(in_progress passing failing)).freeze
```